### DISCUSSION TITLE
Bad voice attribute

### DISCUSSION THREAD

**Discussion by Necronomicron** [espeak-ng/espeak-ng#1456] [Q&A]
Hello! I can't make it work with MBROLA voices.
I've installed espeak-ng-X64.msi, installed MbrolaTools35.exe from [here](https://api.256file.com/mbrolatools35.exe/m-download-138135.html), because link in the docs is dead as a doornail, but espeak-ng still couldn't find mbrola.dll, so I just downloaded it from [here](https://github.com/thiekus/MBROLA/releases/tag/3.3) and put it next to espeak-ng.exe. Then I've downloaded some MBROLA voices and put them into `c:\Program Files\eSpeak NG\espeak-ng-data\mbrola\` as [stated in the docs](https://github.com/espeak-ng/espeak-ng/blob/master/docs/mbrola.md#windows-installation):
```
Error: The specified espeak-ng voice does not exist.
```
So I had to move it to `c:\Program Files\eSpeak NG\espeak-ng-data\voices\`. But it still doesn't work:
```
C:\>espeak-ng -v en1 cat
Bad voice attribute: MBROLA2.060D
Bad voice attribute: A:
Unknown phoneme table: ''

C:\>espeak-ng -v mb-en1 cat
Error: Could not load the specified mbrola voice file.
Error: The specified espeak-ng voice does not exist.
```

Also, is there an IPA to espeak-ng table in the docs? I've found a [table for English vowels](https://github.com/espeak-ng/espeak-ng/blob/master/docs/languages/gmw/en.md) and [another table for English sounds](https://espeak.sourceforge.net/phonemes.html). I've found some table in [another project](https://github.com/rhasspy/gruut-ipa#Phones), but not here.

**Comment by alex19EP** [comment]
hmm. I am not sure if we building mbrola support on windows.

**Reply by Necronomicron** [reply]
Why is it in the docs then?
